src/prognostic_equations/edmfx_closures.jl

```diff
@@ -218,10 +218,13 @@ function mixing_length(
     # kz scale (surface layer)
     if obukhov_length < 0.0 #unstable
         l_W =
-            vkc * (ᶜz - z_sfc) / (sqrt(sfc_tke / ustar / ustar) * c_m) *
+            vkc * (ᶜz - z_sfc) /
+            max(sqrt(sfc_tke / ustar / ustar) * c_m, eps(FT)) *
             min((1 - 100 * (ᶜz - z_sfc) / obukhov_length)^FT(0.2), 1 / vkc)
```

Comment:
What is the 'min' factor here? I do not understand where it comes from. At least we need documentation (e.g., is the exponent 0.2 universal or some ad hoc factor)?

Reply:
This is copied from TurbulenceConvection: https://github.com/CliMA/TurbulenceConvection.jl/blob/b613a8f7c9d6336ac361f4c76edfeb6dbfb51361/src/closures/mixing_length.jl#L20. I didn't find details in Ignacio's paper, except that it says an "empirical stability function" after eq. 34. Maybe we should use the one consistent with SurfaceFluxes?

Reply:
Ok. It looks like Businger functions. It would be best to get these from SurfaceFluxes.

Reply:
Sounds good, I don't want to change the calculation of mixing length in this PR, but I'll change this next.